test/common/http/http3_status_tracker_test.cc

```diff
@@ -0,0 +1,181 @@
+#include "common/http/http3_status_tracker.h"
+
+#include "test/mocks/common.h"
+#include "test/mocks/event/mocks.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using Envoy::Event::MockTimer;
+using testing::NiceMock;
+using testing::Return;
+using testing::StrictMock;
+
+namespace Envoy {
+namespace Http {
+
+namespace {
+class Http3StatusTrackerTest : public testing::Test {
+public:
+  Http3StatusTrackerTest()
+      : timer_(new StrictMock<MockTimer>(&dispatcher_)), tracker_(dispatcher_) {}
```

Comment:
I think we usually omit the StrictMock part since by default all the mocks are strict (we're not super consistent here so not a big deal)

Reply:
Wow, interesting! What's the magic that makes all the Mocks strict? Some #define something somewhere? Very cool. In any case, done.

Reply:
This bit: https://github.com/envoyproxy/envoy/blob/main/bazel/envoy_test.bzl#L170-L172